appveyor.yml

```diff
@@ -5,24 +5,28 @@ image:
 - Visual Studio 2019
 - Visual Studio 2017
 
+branches:
+  only:
+    - master
+
 # Set up environment variables for build info
 environment:
   CUBERITE_BUILD_SERIES_NAME: AppVeyor
   CUBERITE_BUILD_ID: "%APPVEYOR_BUILD_NUMBER%"
   CUBERITE_BUILD_DATETIME: "%APPVEYOR_REPO_COMMIT_TIMESTAMP%"
 
   matrix:
-  - job_name: Windows-x64-debug
+  - job_name: Windows-x64-Debug
```

Comment:
@mathiascode do we distribute the debug build?

Reply:
Not officially.

Reply:
I guess changing this is okay then.